src/main/java/net/imglib2/img/display/imagej/ImageJVirtualStack.java

```diff
@@ -92,58 +99,57 @@ protected ImageJVirtualStack(final RandomAccessibleInterval< S > source, final C
 	{
 		super( (int) source.dimension( 0 ), (int) source.dimension( 1 ), null, null );
 
-		this.source = source;
+		// if the source interval is not zero-min, we wrap it into a view that
+		// translates it to the origin
+		// if we were given an ExecutorService, use a multithreaded projector
+		this.source = zeroMin( source );
 		assert source.numDimensions() > 1;
 
 		int tmpsize = 1;
 		for ( int d = 2; d < source.numDimensions(); ++d )
 			tmpsize *= (int) source.dimension( d );
 		this.size = tmpsize;
 
-		final int sizeX = (int) source.dimension( 0 );
-		final int sizeY = (int) source.dimension( 1 );
-
-		img = new ArrayImgFactory<>( type ).create( new long[] { sizeX, sizeY } );
-
 		higherSourceDimensions = new long[3];
 		higherSourceDimensions[0] = ( source.numDimensions() > 2 ) ? source.dimension( 2 ) : 1;
 		higherSourceDimensions[1] = ( source.numDimensions() > 3 ) ? source.dimension( 3 ) : 1;
 		higherSourceDimensions[2] = ( source.numDimensions() > 4 ) ? source.dimension( 4 ) : 1;
 		this.numDimensions = source.numDimensions();
+		this.type = type;
 
-		// if the source interval is not zero-min, we wrap it into a view that
-		// translates it to the origin
-		// if we were given an ExecutorService, use a multithreaded projector
-		RandomAccessible< S > zeroMin = zeroMin( source );
-		this.projector = ( service == null )
-				? new IterableIntervalProjector2D<>( 0, 1, zeroMin, img, converter )
-				: new MultithreadedIterableIntervalProjector2D<>( 0, 1, zeroMin, img, converter, service );
 
+		this.converter = converter;
 		this.service = service;
-
 		this.bitDepth = initBitDepth( ijtype );
-		this.imageProcessor = initImageProcessor( ijtype, sizeX, sizeY );
 	}
 
-	private RandomAccessible< S > zeroMin( RandomAccessibleInterval< S > source )
+	protected void setMinAndMax( double min, double max ) {
+		this.min = min;
+		this.max = max;
+	}
+
+	private static < S > RandomAccessibleInterval< S > zeroMin( RandomAccessibleInterval< S > source )
 	{
-		return Views.isZeroMin( source ) ? source : Views.zeroMin( source );
+		if ( Views.isZeroMin( source ) )
+			return source;
+		Interval zeroMinInterval = new FinalInterval( Intervals.dimensionsAsLongArray( source ) );
+		return Views.interval( Views.zeroMin( source ), zeroMinInterval );
```

Comment:
This (`Views.interval`) is unnecessary. `Views.zeroMin` already does it. It was fine before.
Actually, the whole `return Views.isZeroMin( source ) ? source : ...;` should be done in `Views.zeroMin()`, so that we can get rid of this zeroMin method here altogether